packages/teams-js/src/public/tasks.ts

```diff
@@ -36,8 +41,11 @@ export namespace tasks {
         (sdkResponse: dialog.ISdkResponse) => submitHandler(sdkResponse.err, sdkResponse.result)
       : undefined;
     if (taskInfo.card !== undefined || taskInfo.url === undefined) {
-      ensureInitialized(FrameContexts.content, FrameContexts.sidePanel, FrameContexts.meetingStage);
-      sendMessageToParent('tasks.startTask', [taskInfo as DialogInfo], submitHandler);
+      if (taskInfo.completionBotId) {
+        dialog.adaptiveCard.bot.open(getBotAdaptiveCardDialogInfoFromTaskInfo(taskInfo), dialogSubmitHandler);
```

Comment:
Since adaptive card is a sub capability and hosts can choose to not implement it, should we make a check here before we call dialog.adaptiveCard.xxx?

Reply:
I think that makes sense.